docs/DeveloperGuide.md

```diff
@@ -212,6 +212,8 @@ Below is an activity diagram summarising the possible paths for an edit command:
     * Pros: Allows user to revert their changes
     * Cons: System crashes will not save the edits.
 
+We chose to use alternative 1 in our implementation because in manually using the application and assessing its performance, we found that saving after every edit did not make a meaningful difference to performance and would not affect user experience.
```

Comment:
```suggestion
We chose to use alternative 1 in our implementation because in manually using the application and assessing its performance, we found that manually saving after every edit did not make a meaningful difference to performance and would not affect user experience.
```